.github/workflows/android.yml

```diff
@@ -0,0 +1,127 @@
+name: "Android : Build & Upload to Firebase App Distribution 🤖"
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+on: [push]
```

Comment:
Do we plan to deploy the app on any push or for the push into the main branch?

Reply:
This is to the firebase, that means for each PR we will get a new deploy to the firebase (if more commits are added to the same PR, that specific apk is overwritten, this gives us the flexibility to have multiple apks if we have several PRs)

Reply:
we can use same `pull_request` setting